html/elements/col.json

```diff
@@ -115,7 +115,7 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": false,
+                "version_added": "≤72",
                 "impl_url": "https://bugzil.la/2212"
```

Comment:
Should this PR be kept? I'm not sure.

Reply:
I'll be revisiting this soon, there's a lot of elements with the `char` and `charoff` attributes!